types/mparticle__web-sdk/index.d.ts

```diff
@@ -309,6 +355,23 @@ interface GetSession {
     (): string;
 }
 
+// Rokt Method Interfaces
+interface AttachKit {
+    (kit: RoktKit): void;
+}
```

Comment:
I don't think we need to include this because developers won't actually use it.  it's only internal